notmuch/private.h

```diff
@@ -40,6 +40,7 @@ struct Mailbox;
    (LIBNOTMUCH_MAJOR_VERSION == (major) &&                                        \
     LIBNOTMUCH_MINOR_VERSION == (minor) && LIBNOTMUCH_MICRO_VERSION >= (micro)))
 
+extern const char NmUrlProtocol[];
```

Comment:
Does it need to be extern?

Reply:
My understanding is that it does since it's used in `notmuch/notmuch.c` and now `notmuch/config.c`. Am I missing something obvious?

Reply:
No, I was :)